vendor/engines/split_accounts/app/services/split_accounts/converters/order_detail_to_journal_rows.rb

```diff
@@ -0,0 +1,52 @@
+module SplitAccounts
+  module Converters
+    class OrderDetailToJournalRows
+
+      SplitAmount = Struct.new(:split, :amount)
+
+      attr_accessor :journal, :order_detail, :total, :splits
+
+      def initialize(journal, order_detail, options = {})
+        @journal = journal
+        @order_detail = order_detail
+        @total = options[:total] || order_detail.total
```

Comment:
They help with testing.  But primarily I need to be able to override the `total` from a root converter to a leaf converter.  A converter should be able to pass a new total to another converter anywhere throughout the converter chain.  This way we could go as fas as to support conversions like `Split Account 1 => Split Account 2 => NufsAccount (the repo specific version)`.  Does that make sense?